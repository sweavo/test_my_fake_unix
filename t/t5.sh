#!/bin/bash
## In MSYS1, $() within a for expression does not propagate its return status
## to the outer shell.  I don't know if this is the case in all bashes.

function exit_handler 
{
    echo exit_status $?
    echo done
}
trap exit_handler EXIT

set -e # exit on error

source paths.source

(
    for x in $(wef bibble); do echo "hi"; done
    echo outer_bash_got: $?
)

