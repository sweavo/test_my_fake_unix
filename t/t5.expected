t/t5.sh: line 19: wef: command not found
outer_bash_got: 127
exit_status 0
done
